### ISSUE TITLE
CSE: Interop will not initialize

### ISSUE THREAD

**Issue by ShimShon48** [shadeMe/Construction-Set-Extender#10]
Under CSE, error received when attempting to generate a lip file against a mp3 data file and text.

**Comment by shadeMe** [comment]
The current build has a bug that causes the very first LIP file generation of a session to fail - It'll start working thereafter. A fix for this bug is coming in the next update.

**Comment by ShimShon48** [comment]
I have missed our technical discussions. I know C and C++ and can review code. If you would like me to donate to your CSE development effort, send me your Paypal information. This is Keith Woodward; you made the Mod KoS.

**Comment by ShimShon48** [comment]
I volunteer to be one of your beta testers.

**Comment by shadeMe** [comment]
I periodically upload new builds to the development channel (the link can be found in the Description/Mirrors section of the [Nexus page](https://www.nexusmods.com/oblivion/mods/36370?tab=description)). 

Re. donations - Appreciate the thought :) There should be a link somewhere on my Nexus profile if you feel inclined, but it's definitely not necessary.

**Comment by ShimShon48** [comment]
You latest build uploaded to Nexus was 13 October 2021

**Comment by ShimShon48** [comment]
Friend,

 

I have a quick question that has been plaguing my Mod almost at the beginning of its development. I have created a essentially a patch for most of the Oblivion world of Tamiel. What is happening is a game crash immediately after entering the Oak and Crosier. I have used TES4Edit to clean my Mod regularly and am using the highest version of OBSE available. I modded the Oak and Crosier and created a new Talasma NPC with an unique Publican script. I modded all of Cyrodil in respect to the city. I have been very careful not to corrupt anything. I even created a trap door to the O&C cellar for a back door entrance into the Tavern. It still crashes upon entering. 

 

Keith
